cli/util/options.js

```diff
@@ -238,7 +238,7 @@ function merge(config, currentOptions, parentOptions, parentBaseDir) {
 exports.merge = merge;
 
 const dynrequire = typeof __webpack_require__ === "function"
-  ? __non_webpack_require__
+  ? () => module[`require`] // backticks required
   : require;
```

Comment:
```js
const dynrequire = typeof __webpack_require__ === "function"
  ? typeof module !== 'undefined' && 
    typeof module.require === 'function' && 
    module.require.bind(module)
  : require;
```

Reply:
Would that work with ESM (no `module`)?

Reply:
I don't know

Reply:
Unfortunately `module.require` is not `require`. The latter is a wrapper around the former, and only this latter provides the `.resolve()` method that you are relying on.

I give up. Besides using `eval('require')` (with a try/catch), I cannot find a way to make it work.
But that's a problem, because the library cannot be included in a Webpack project without dirty tricks (such as forcing webpack to ignore `ts-node`).

Do you really need `require.resolve`? I need to delve into how it is used by options.js and asc.js. Since we are sure this call will be done by a Node environment, we could do the same with `fs` maybe.